code/stylesheet.dm

```diff
@@ -92,6 +92,7 @@ h1.alert, h2.alert		{color: #000000;}
 .russian				{color: #635d17}
 .serbian				{color: #Ad6f1d}
 .jive					{color: #666666}
+.german					{color: #9933FF}
```

Comment:
Spacings a bit off

Reply:
strange, that wasnt there before... i will check up the code, and i was told to remove the former code that hat id. When i added known_languages to mob in Oberth, it didnt compile the code with travis
I will check out the mercenary.dm